docs/core-features/extensions.md

```diff
@@ -541,15 +533,15 @@ assert circuit.encrypt_run_decrypt(5) == 5
 ```
 
 {% hint style="info" %}
-Identity extension can be used to clone an input while changing its bit-width. Imagine you have `return x**2, x+100` where `x` is 2-bits. Because of `x+100`, `x` will be assigned 7-bits and `x**2` would be more expensive than it needs to be. If `return x**2, fhe.identity(x)+100` is used instead, `x` will be assigned 2-bits as it should and `fhe.identity(x)` will be assigned 7-bits as necessary.
+The `fhe.identity` extension is useful for cloning an input with a different bit-width.
+- Imagine you have `return x**2, x+100` where `x` is 2-bits. Because of `x+100`, `x` will be assigned 7-bits and `x**2` would be more expensive than it needs to be. If `return x**2, fhe.identity(x)+100` is used instead, `x` will be assigned 2-bits as it should and `fhe.identity(x)` will be assigned 7-bits as necessary.
```

Comment:
do we want to have a `-` here?

Reply:
I think it's better, otherwise the paragraph is too long to read and lacks focus.